edbdeploy/data/ansible/roles/bdr_pot_setup/tasks/pg_user_pgpass.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: Generate the pg_project_user_password
+  include_role:
+    name: edb_devops.edb_postgres.manage_dbserver
+    tasks_from: generate_password
+  vars:
+    input_user: "{{ pg_project_user }}"
+    input_password: "{{ pg_project_user_password }}"
+  no_log: "{{ disable_logging }}"
+  when: pg_project_user_password|length < 1
+
+- name: Add the pgpass for os_project_user on the servers
+  include_role:
+    name: edb_devops.edb_postgres.manage_dbserver
+    tasks_from: manage_pgpass
+  vars:
+    pg_pgpass_values:
+      - user: "{{ pg_project_user }}"
+        password: "{{ input_password }}"
```

Comment:
If `pg_project_user_password` has been defined then `input_password` is not defined.
You should add a task before this one, merging `input_password` and `pg_project_user_password`.

Reply:
input_password gets updated by the following:
```
- name: Generate the pg_ansible_user_password
  include_role:
    name: edb_devops.edb_postgres.manage_dbserver
    tasks_from: generate_password
  vars:
    input_user: "{{ pg_ansible_user }}"
    input_password: "{{ pg_ansible_user_password }}"
  no_log: "{{ disable_logging }}"
  when: pg_ansible_user_password|length < 1
```
So, I don't we need to use anything for merging input_password.

Reply:
But, if `pg_ansible_user_password` is set (length > 0), `input_password` won't be set, right?

Reply:
Ah! I see. Fixed.